src/main_app/db/services/admin_service.py

```
@@ -11,108 +11,81 @@
+        """Toggle coordinator activity."""
+        # record = get_coordinator_by_id(coordinator_id)
+        try:
+            record = self.session.query(AdminUserRecord).filter(AdminUserRecord.id == coordinator_id).first()
+            if not record:
+                return None
+
+            record.is_active = is_active
+            self.session.commit()
+            self.session.refresh(record)
+            return record
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)